packages/app-admin-ui/client/pages/List/index.js

```diff
@@ -247,6 +246,12 @@ const ListPage = props => {
     }
   }, []);
 
+  useEffect(() => {
+    if (list.adminConfig.singleton && itemCount > 0) {

```

Comment:
again, should this be a different naming to redirect to first item? should we do that as generic feature?